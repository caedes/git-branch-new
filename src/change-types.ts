export interface ChangeType {
  name: string;
  description: string;
}

export const changeTypes: ChangeType[] = [
  {
    name: "test",
    description: "Adding missing tests",
  },
  {
    name: "feat",
    description: "A new feature",
  },
  {
    name: "fix",
    description: "A bug fix",
  },
  {
    name: "chore",
    description: "Build process or auxiliary tool changes",
  },
  {
    name: "docs",
    description: "Documentation only changes",
  },
  {
    name: "refactor",
    description: "A code change that neither fixes a bug or adds a feature",
  },
  {
    name: "style",
    description: "Markup, white-space, formatting, missing semi-colons...",
  },
  {
    name: "ci",
    description: "CI related changes",
  },
  {
    name: "perf",
    description: "A code change that improves performance",
  },
];
