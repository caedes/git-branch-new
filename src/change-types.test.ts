import { describe, expect, it } from "vitest";

import { changeTypes } from "./change-types";

describe("changeTypes", () => {
  it("exists", () => {
    expect(changeTypes).toMatchSnapshot();
  });
});
