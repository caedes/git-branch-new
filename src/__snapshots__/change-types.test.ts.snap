// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`changeTypes > exists 1`] = `
[
  {
    "description": "Adding missing tests",
    "name": "test",
  },
  {
    "description": "A new feature",
    "name": "feat",
  },
  {
    "description": "A bug fix",
    "name": "fix",
  },
  {
    "description": "Build process or auxiliary tool changes",
    "name": "chore",
  },
  {
    "description": "Documentation only changes",
    "name": "docs",
  },
  {
    "description": "A code change that neither fixes a bug or adds a feature",
    "name": "refactor",
  },
  {
    "description": "Markup, white-space, formatting, missing semi-colons...",
    "name": "style",
  },
  {
    "description": "CI related changes",
    "name": "ci",
  },
  {
    "description": "A code change that improves performance",
    "name": "perf",
  },
]
`;
